src/app/dashboard/list/list.component.html

```diff
@@ -0,0 +1,32 @@
+<mat-list
+  cdkDropList
+  (cdkDropListDropped)="dropTasks($event)"
+  [cdkDropListData]="tasks"
+  class="task-list"
+>
+  <mat-list-item class="task-list-item" *ngFor="let task of tasks" cdkDrag>
+    <app-task
+      (toggleTask)="updateTaskStatus($event)"
+      (deleteTask)="deleteTask($event)"
+      (renameTask)="renameTask($event)"
+      [task]="task"
+    ></app-task>
+  </mat-list-item>
+</mat-list>
+<form [formGroup]="taskNameGroup">
```

Comment:
Adding "group" at the end of the form group name is redundant. We want to avoid redundancy in our naming especially if we already have the context (in this case, we're passing a `taskNameGroup` to a property called `formGroup`). Same for `taskNameControl`.

As an example, the form group should be named: `newTask` or `newTaskForm`, and the control just: `name`

Please implement this where applicable.